CHANGES

```diff
@@ -1,3 +1,13 @@
+SUMMARY OF RELEASE CHANGES FOR OPENVAS-SMB
+==========================================
+
+For detailed code changes, please visit
+ https://github.com/greenbone/openvas-smb/commits/master
+or get the entire source code repository and view log history:
+$ git clone git@github.com:greenbone/openvas-smb.git
```

Comment:
Please use
  `git clone https://github.com/greenbone/openvas-smb.git`
instead.

Otherwise git will try to clone the repository via SSH which will fail for people without GitHub account. HTTPS is also less likely to be blocked by network infrastructure.